scripts/Start.ps1

```diff
@@ -8,6 +8,41 @@ $FrontendScript = Join-Path "$PSScriptRoot" 'Start-Frontend.ps1'
 
 # Start backend (in new PS process)
 Start-Process pwsh -ArgumentList "-command $BackendScript"
+# check if the backend is running before proceeding
+$backendRunning = $false
 
-# Start frontend (in current PS process)
-& $FrontendScript
+# get the port from the REACT_APP_BACKEND_URI env variable
+$envFilePath = Join-Path $PSScriptRoot '..\webapp\.env'
+$envContent = Get-Content -Path $envFilePath
+$port = [regex]::Match($envContent, ':(\d+)/').Groups[1].Value
+
+# set max retries to 5
+$maxRetries = 5
+
+# set retry count to 0
```

Comment:
This comment should be removed: it just repeats what's self evident from the code